### PR TITLE
[digitalSTROM] Fix possible NPE's

### PR DESCRIPTION
**Pull request by hmerk**
Fixes https://github.com/eclipse-archived/smarthome/issues/6488 which was not transfered to openHAB-addons

Signed-off-by: Hans-Jörg Merk <github@hmerk.de>

### CODE REVIEW

File: bundles/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/handler/BridgeHandler.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 import java.util.concurrent.TimeUnit;
 
 import org.apache.commons.lang.StringUtils;
+import org.eclipse.jdt.annotation.NonNull;
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/handler/BridgeHandler.java:29]: `NonNull` annotations should not be needed. It means the `NonNullByDefault` on class level is missing, which is the preferred way to annotate. I can image it's a lot more work to add `NonNullByDefault` to the class, so I don't expect to add that, but I also don't think you should add the `NonNull` annotations as a kind of half fix. So it's better to have no annotations than to have a wrong half solution.

**Comment by hmerk** [comment]
Thanks @Hilbrand for your valuable comment. I have removed the annotations again.

File: bundles/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/handler/BridgeHandler.java (hunk 1 above)
Comment (hmerk) [reply bundles/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/handler/BridgeHandler.java]: Removed

**Review by Hilbrand — APPROVED** [review]
LGTM

**Pull request merged**